### PR TITLE
Allow to use ../theme in portal and 6.x forms homepages

### PR DESCRIPTION
**Pull request by abirembaut**

### CODE REVIEW

File: common/src/main/java/org/bonitasoft/console/common/server/servlet/ResourceServlet.java [hunk 1]
```diff
@@ -82,9 +84,18 @@
...
+        if (pathInfo != null && pathInfo.startsWith("/") && pathInfo.length() > 1) {
+            //Support relative calls to the THEME from the homepage using ../theme
+            fileName = pathInfo.substring(1);
+        } else {
+            fileName = request.getParameter(LOCATION_PARAM);
+        }
+        String resourceName = request.getParameter(getResourceParameterName());
+        if (resourceName == null) {
+            resourceName = getDefaultResourceName();
+        }
```
Comment (velcrin) [inline common/src/main/java/org/bonitasoft/console/common/server/servlet/ResourceServlet.java]: How do you feel about adding a test? Manageable?

Comment (abirembaut) [inline common/src/main/java/org/bonitasoft/console/common/server/servlet/ResourceServlet.java]: yep. probably
Let me check

Comment (abirembaut) [inline common/src/main/java/org/bonitasoft/console/common/server/servlet/ResourceServlet.java]: done and fixed

File: common/src/test/java/org/bonitasoft/console/common/server/servlet/ResourceLocationReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    @Mock
+    private HttpServletRequest req;
+
+    @Spy
+    private final ResourceLocationReader resourceLocationReader = new ResourceLocationReader();
+
+    @Test
+    public void should_use_infopath_if_location_param_is_not_defined() throws Exception {
+
+        when(req.getMethod()).thenReturn("GET");
```
Comment (velcrin) [inline common/src/test/java/org/bonitasoft/console/common/server/servlet/ResourceLocationReaderTest.java]: I don't think you need to stub that

File: common/src/test/java/org/bonitasoft/console/common/server/servlet/ResourceLocationReaderTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        final String resourceLocation = resourceLocationReader.getResourceLocationFromRequest(req);
+
+        verify(req, times(1)).getPathInfo();
+        assertThat(resourceLocation).isEqualTo("bonita.css");
+    }
+
+    @Test
+    public void should_not_use_infopath_if_location_param_is_defined() throws Exception {
+
+        when(req.getMethod()).thenReturn("GET");
```
Comment (velcrin) [inline common/src/test/java/org/bonitasoft/console/common/server/servlet/ResourceLocationReaderTest.java]: Here neither

File: common/src/test/java/org/bonitasoft/console/common/server/themes/ThemeResourceServletTest.java [hunk 4]
```diff
@@ -58,6 +71,46 @@ public void should_verify_authorisation_for_the_given_location_param() throws
         }
     }
 
+    @Test
+    public void should_use_infopath_if_location_param_is_not_defined() throws Exception {
```
Comment (velcrin) [inline common/src/test/java/org/bonitasoft/console/common/server/themes/ThemeResourceServletTest.java]: Do we still need those tests now that this behaviour does not belong to `ThemeResourceServlet`anymore?

**Pull request merged**